bats/fb-katello-proxy.bats

```diff
@@ -23,6 +23,8 @@ setup() {
 }
 
 @test "sync proxy" {
+  # workaround for https://projects.theforeman.org/issues/32275
+  hammer setting set --name check_services_before_actions --value false
```

Comment:
Should this be behind a version check?

Reply:
honestly, it shouldn't be here, at all. I hope we can just fix that for 4.0 GA :)